### PR TITLE
Developer

### PR DESCRIPTION
**Pull request by Krupartha**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -1,17 +1,34 @@
...
     data = str(request.form['words'])
     data1 = TextBlob(data)
+    print(data1)
     return str(data1.correct())
 
+@app.route('/ir', methods = ['GET','POST'])
+def irish():
+    res =""
+    irish_words = (request.form['words']).split(" ")
+    for irish_word in irish_words:
```
Comment (hpakala1) [inline app.py]: When executed and after typing a word it is displaying two results

Comment (Krupartha) [reply app.py]: OK I will correct the issue

**Review by Krupartha — COMMENTED** [review]
Everything looks good now

File: mainpage [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<body>
+	<script src="https://code.jquery.com/jquery-3.6.0.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>
+<h3>Example of auto spell correction</h3>
+		<textarea id="words" ></textarea>
+		<button onClick="submit()">Submit</button>
+<script>
+	function submit(){
+	$.post("http://192.168.0.100:5000/", {words:$("#words").val()}, function(result){
```
Comment (smedipally) [inline mainpage]: Connect both the english and irish html pages

File: english (1).html [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+          if( ui.item.value === "no suggestions found"){
+              return false
+          }
+          var terms = split( this.value );
+          terms.pop();
+          terms.push( ui.item.value );
+          terms.push( "" );
+          this.value = terms.join( " " );
+          return false;
+        }
```
Comment (Krupartha) [inline english (1).html]: Add Space function

File: english (1).html [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+        </ul>
+        <br/>
+        <br/>
+            <textarea class="form-control" id="tags">
+            </textarea>
+            <br/>
+            <button onClick="submit()" class="btn btn-default">Verify</button>   
+      </div>
+      <script>
+        function submit(){
```
Comment (Krupartha) [inline english (1).html]: Remove the submit button

File: mainpage [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+<head>
+  <title>Spell Checker</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+</head>
+<body>
+
```
Comment (mukthipola345) [inline mainpage]: Updated

**Review by mukthipola345 — COMMENTED** [review]
Changes done

File: english (1).html (hunk 3 above)
Comment (hpakala1) [reply english (1).html]: Added

File: english (1).html (hunk 4 above)
Comment (hpakala1) [reply english (1).html]: Removed it

File: Irish.html [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+</head>
+<body>
+    <div class="container">
+        <h2>Spell Checker</h2>
+        <br/>
+        <ul class="nav nav-pills">
+          <li ><a href="english.html">English</a></li>
+          <li class="active"><a href="Irish.html"> Irish </a></li>
+        </ul>
```
Comment (mukthipola345) [inline Irish.html:27]: Where are the irish suggestions?

File: English.html [hunk 7]
```diff
@@ -0,0 +1,223 @@
...
+  <script src="https://code.jquery.com/jquery-3.6.0.js"></script>
+  <script src="https://code.jquery.com/ui/1.13.0/jquery-ui.js"></script>
+  <script type="text/javascript" src="./words.js"></script>
+<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>  
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+</head>
+<body>
+    <div class="container">
```
Comment (Krupartha) [inline English.html:21]: How will I get the english suggestions?

File: Irish.html (hunk 6 above)
Comment (smedipally) [reply Irish.html:27]: Yes,its done I have updated the code.

File: English.html [hunk 8]
```diff
@@ -0,0 +1,223 @@
...
+      .on( "change",
+       function( event ) {
+        if ( event.keyCode === $.ui.keyCode.TAB &&
+            $( this ).autocomplete( "instance" ).menu.active ) {
+          event.preventDefault();
+        }
+      }
+      )
+      .autocomplete({
+        minLength: 0,
```
Comment (smedipally) [inline English.html:65]: Is this user friendly?I mean that there is no description about how to operate the application?

File: English.html (hunk 7 above)
Comment (hpakala1) [reply English.html:21]: I have updated it.

File: English.html (hunk 8 above)
Comment (hpakala1) [reply English.html:65]: Now you can understand it easily.

File: Index [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+<head>
+  <title>Spell Checker</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+</head>
+<body>
+
```
Comment (Krupartha) [inline Index:12]: I think the user must understand what to do in this page

Comment (mukthipola345) [reply Index:12]: Done,Now everyone can understand it very simply.

**Review by mukthipola345 — COMMENTED** [review]
Now,the project looks well now.

**Review by Krupartha — COMMENTED** [review]
Well done,the code is working.

**Pull request merged**